testsuite/features/step_definitions/common_steps.rb

```diff
@@ -89,9 +89,13 @@
   step %(I should see a "#{ram_mb}" text)
 end
 
-When(/^I check the MAC address value$/) do
-  get_mac_address = 'cat /sys/class/net/eth0/address'
-  mac_address, _local, _remote, _code = $client.test_and_store_results_together(get_mac_address, 'root', 600)
+When(/^I check the MAC address value on "([^"]*)"$/) do |host|
+  node = get_target(host)
+  get_system_id_like = "awk -F= '/^ID_LIKE/{print $2}' /etc/os-release"
+  id_like, _local, _remote, _code = node.test_and_store_results_together(get_system_id_like, 'root', 600)
```

Comment:
Forget this `id_like` thing, the `host` will tell you directly if debian or not. Just test the value of this variable, no need to ssh.

Reply:
Ok, as we only have ubuntu-minion, it's fine.